app/helpers/application_helper.rb

```diff
@@ -1,7 +1,21 @@
 require 'csv'
 require 'json'
+require 'open3'
 
 module ApplicationHelper
+  def sanitize(user_input_text)
+    # accept alpanumeric characters only
+    user_input_text.gsub(/[^A-Za-z0-9]/, '-')
```

Comment:
for the regex, we still want to accept spaces in the project names right? I think we can include space. As long as the params are separated Open3 will handle the spaces apparently:

```
irb(main):010:0> stdin, stdout, stderr = popen3("aws", "s3", "ls", "s3://idseq-samples-staging && echo 'HELLO'")
=> [#<IO:fd 18>, #<IO:fd 20>, #<IO:fd 22>, #<Process::Waiter:0x00007fa44f14e4c0 sleep>]
irb(main):011:0> puts stdout.read

=> nil
irb(main):012:0> stderr.read
=> "\nParameter validation failed:\nInvalid bucket name \"idseq-samples-staging && echo 'HELLO'\": Bucket name must match the regex \"^[a-zA-Z0-9.\\-_]{1,255}$\"\n"
irb(main):013:0>
```

Reply:
That's true. I will make the regex more permissive

Reply:
thanks

Reply:
"_" and "-" too probably